katdal/chunkstore_s3.py

```diff
@@ -467,6 +483,10 @@ def create_array(self, array_name):
 
         if self.expiry_days > 0:
             xml_payload = _BASE_LIFECYCLE_POLICY.format(self.expiry_days)
+            if self.validate_xml_policies:
+                if not schemas.has_lxml:
+                    raise ImportError("XML schema validation requires lxml to be installed.")
```

Comment:
It would be nice to link to the original error via `raise_from`. See e.g. the RDB reader handling in katsdptelstate.

Reply:
Not sure I agree - I prefer the very unambiguous nature of just telling the user that you need lxml if you want to use validators.